install.py

```diff
@@ -1031,5 +1039,13 @@ def yes_or_no(value):
     return ('yes' if value else 'no')
 
 
+def search_for(path, filename):
+    for r,d,f in os.walk(path):
```

Comment:
Please adhere to the coding conventions and in particular PEP-8 (variable names should be >= 3-letters long):
```suggestion
    for rootdir, dirs, filenames in os.walk(path):
```